src/value/value.rs

```diff
@@ -353,9 +353,9 @@ macro_rules! impl_from_array {
 
 impl_from_array!(1, 2, 3, 4, 5, 6, 7, 8);
 
-impl<'a> From<&'a str> for Value {
-    fn from(value: &'a str) -> Value {
-        Value::String(Tag::Default, value.to_string().into())
+impl From<&str> for Value {
+    fn from(value: &str) -> Value {
+        Value::String(Tag::Default, value.to_string())
```

Comment:
This was not actually a clippy suggestion, I saw myself that the lifetime could be elided here.